app/assets/stylesheets/pages/home/_ukm.scss

```diff
@@ -0,0 +1,158 @@
+.ukm{
+	margin-top: 33px;
```

Comment:
try to measure it with the ruler extension, recheck the gap from the mockup

Reply:
uh she up